chainerx_cc/chainerx/array_body_leak_detection.cc

```diff
@@ -55,12 +55,12 @@ bool ArrayBodyLeakTracker::IsAllArrayBodiesFreed(std::ostream& os) const {
     return true;
 }
 
-ArrayBodyLeakDetectionScope ::ArrayBodyLeakDetectionScope(ArrayBodyLeakTracker& tracker) {
```

Comment:
Even though these fixes seem appropriate, they're irrelevant to this PR.
Could you omit those fixes from this PR? (Just one or two might be okay but it's too many).

Reply:
Sure, I'll revert these changes.